src/vllm_tgis_adapter/grpc/grpc_server.py

```diff
@@ -865,14 +859,25 @@ async def Tokenize(
         # TODO: maybe parallelize, also move convert_ids_to_tokens into the
         # other threads
         for req in request.requests:
-            batch_encoding = tokenizer.encode_plus(
-                text=req.text,
-                return_offsets_mapping=request.return_offsets,
-                add_special_tokens=ADD_SPECIAL_TOKENS,
-            )
+            if not hasattr(tokenizer, "encode_plus"):
+                if request.return_offsets:
+                    raise ValueError(
+                        f"{type(tokenizer)} doesn't support "
+                        "return_offsets at the moment. "
```

Comment:
suggested wording adjustment
```suggestion
                        "the return_offsets option"
```